docker/README.md

```diff
@@ -0,0 +1,54 @@
+# Docker Build Environment
+
+This directory contains Docker-related files for building StarryOS in a consistent Linux environment.
+
+- **Tip**: When running QEMU inside Docker, you may need extra options for hardware acceleration and networking (for example, `--device /dev/kvm` and `--cap-add=NET_ADMIN`). For more details, please refer to the official QEMU / Docker documentation.
+
+## Files
+
+- `Dockerfile` - Main Docker image definition
+- `docker-compose.yml` - Optional Docker Compose configuration (for convenience)
+
+## Usage
+
+### Using Docker directly (Recommended)
+
+From the project root:
+
+```bash
+# Build the Docker image
+$ docker build -t starryos:latest -f docker/Dockerfile .
+
+# Run the container (Unix/macOS/Linux/Windows PowerShell)
+$ docker run -it --rm -v ${PWD}:/workspace -w /workspace starryos:latest bash
+
+# For Windows Command Prompt:
+> docker run -it --rm -v %cd%:/workspace -w /workspace starryos:latest bash
```

Comment:
I think no one will use `cmd` for developing. If you insist to keep this part, I want to know if `.` syntax works here.